config/routes.rb

```diff
@@ -39,7 +39,7 @@
     get 'location', on: :new
   end
 
-  resources :organisations, only: %i[show edit update]
+  resources :organisations, only: %i[edit update]
```

Comment:
Do we need to keep the show action for the super admins?

Reply:
All super admin routes are nested under `namespace: :admin`